src/robot-core/dc-motor-two.cpp

```diff
@@ -0,0 +1,95 @@
+
+/*******************************************************************************
+*                               Standard Libraries
+*******************************************************************************/
+
+#include <Arduino.h>
+
+/*******************************************************************************
+*                               Header Files
+*******************************************************************************/
+
+#include "dc-motor-two.h"
+
+/*******************************************************************************
+*                               Static Functions
+*******************************************************************************/
+
+/*******************************************************************************
+*                               Constants
+*******************************************************************************/
+
+/*******************************************************************************
+*                               Structures
+*******************************************************************************/
+
+/*******************************************************************************
+*                               Variables
+*******************************************************************************/
+
+dc_motor_two_t dc_motor_two_array[] =
+{
+    [LEFT_DRIVING_MOTOR] =
```

Comment:
alignment and naming of the `is_intialized` field of these structs can also be changed

Reply:
done